a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -879,3 +901,30 @@ def _mark_lb_as_error(self, vthunder, loadbalancers_list):
         except Exception as e:
             LOG.exception('Failed to set Loadbalancers to ERROR due to '
                           ': {}'.format(str(e)))
+
+
+class UpdateAcosVersionInVthunderEntry(VThunderBaseTask):
+
+    @axapi_client_decorator
+    def execute(self, vthunder, loadbalancer):
+        compute_flag = False
+        existing_vthunder = self.vthunder_repo.get_vthunder_by_project_id(db_apis.get_session(),
+                                                                          loadbalancer.project_id)
+        if existing_vthunder:
+            compute_flag = self.vthunder_repo.get_delete_compute_flag(db_apis.get_session(),
```

Comment:
Why is this being checked? I don't think we need this right? This just tells us if there is 1 less vthunder entries. But even if there is only one, we can still just use that one's acos_version.

With this code, we end up calling this API twice per project instead of just once.

Reply:
Removed this check.